ludwig/profiling/proto/dataset_profile.proto

```diff
@@ -3,6 +3,8 @@
 //   > sudo apt install protobuf-compiler
 //   > protoc --proto_path=. --python_out=. \
 //       ludwig/profiling/proto/whylogs_messages.proto ludwig/profiling/proto/dataset_profile.proto
+//   > protoc --proto_path=. --go_out=.\
+// .     ludwig/profiling/proto/whylogs_messages.proto ludwig/profiling/proto/dataset_profile.proto
```

Comment:
Is the idea here to write the proto files as both python files and go files?

Reply:
Yes.